server.js

```diff
@@ -985,6 +984,48 @@ routes['join_room'] = function (user, data) {
     room.sync(user);
 }
 
+routes['search_youtube'] = function (user, data) {
+	if(user.room === null) return;
+	var room = user.room;
+
+	var query = encodeURIComponent(data.query);
+
+
+	
+	if (typeof(data.query !== query) || query.length < 1 || query.length > 100) {
```

Comment:
@youresam , you're saying instead of checking for > 100 characters, just trim all queries to 100 characters or less?